### DISCUSSION TITLE
Exporting duplicates to dataset

### DISCUSSION THREAD

**Discussion by tanevitch** [AI-team-UoA/pyJedAI#7] [General]
Hi! 
I would like to know if there is any way to export the duplicates found after the process, instead of listing the metrics result.
Thanks!

**Comment by Nikoletos-K** [comment]
Hello!
Yeah this can be done with the method write here:
https://github.com/AI-team-UoA/pyJedAI/blob/520c2bc2e82c64d322569ec193f1423358e6f4c3/pyjedai/evaluation.py#L216

Cheers🙏
